app/questionnaire/rules.py

```diff
@@ -222,13 +285,21 @@ def get_answer_store_value(answer_index, answer_store, group_instance, group_ins
     return filtered[0]['value'] if filtered.count() == 1 else None
 
 
+def get_answer_store_value_ignoring_group_instance_id(answer_index, answer_store, group_instance):
```

Comment:
I'm not particularly happy with this solution, does anyone have any suggestions?

Reply:
Can we simplify this with `schema.answer_is_in_repeating_group(answer_id)` to decide if we need to include group_instance_id?

Reply:
@dwyeraadam - I had been under the impression that answer_store.filter actually filtered on None values. As Jon said, it doesn't need to, and on further inspection, ignores None values.

So I think this stuff can be completely removed :)

Reply:
I've pulled that out. Ready for review I think